src/ConstrainedPolyModel.cc

```diff
@@ -44,43 +44,26 @@ ConstrainedPolyModel::ConstrainedPolyModel(const CcdImageList &ccdImageList,
   for (auto &ccdImage: ccdImageList)
     {
       const CcdImage &im = *ccdImage;
-      unsigned visit = im.getVisit();
-      unsigned chip = im.getCcdId();
+      auto visit = im.getVisit();
+      auto chip = im.getCcdId();
       auto visitp = _visitMap.find(visit);
       if (visitp == _visitMap.end())
 	{
 	  if (_visitMap.size() == 0)
 	    {
-	      // if one fits all of them, the model is degenerate.
-#ifdef ROTATE_T2
-# warning : hack in ConstrainedPolyModel::ConstrainedPolyModel : rotated frame
-	      _visitMap[visit] = std::unique_ptr<SimpleTransfoMapping>(new SimpleGtransfoMapping(GtransfoLinRot(3.141927/2.), /* ToFit = */ false));
-#else
-	      _visitMap[visit] = std::unique_ptr<SimpleGtransfoMapping>(new SimpleGtransfoMapping(GtransfoIdentity()));
-#endif
+          _visitMap[visit] = std::unique_ptr<SimpleGtransfoMapping>(new SimpleGtransfoMapping(GtransfoLinScale()));
 	      refVisit = visit;
 	    }
-	    else
-#ifdef ROTATE_T2
-	      {
-		GtransfoPoly poly(degree);
-		poly = GtransfoPoly(GtransfoLinRot(3.141927/2.))*poly;
-		_visitMap[visit] = std::unique_ptr<SimplePolyMapping>(new SimplePolyMapping(GtransfoLin(), poly));
-	      }
-#else
-	  _visitMap[visit] = std::unique_ptr<SimplePolyMapping>(new SimplePolyMapping(GtransfoLin(),
-										      GtransfoPoly(degree)));
-#endif
+	    else _visitMap[visit] = std::unique_ptr<SimplePolyMapping>(new SimplePolyMapping(GtransfoLin(),
+                                                                                         GtransfoPoly(degree)));
 	}
       auto chipp = _chipMap.find(chip);
-      if ((chipp == _chipMap.end()) && visit == refVisit )
+      if (chipp == _chipMap.end())
```

Comment:
After deleting the `refVisit` check, I don't think that the variable is needed anymore and can be removed from the function (lines 42 and 55).